### DISCUSSION TITLE
Find&Replace with regexp grouping

### DISCUSSION THREAD

**Discussion by furkanarabaci** [eclipse-theia/theia#9859] [Q&A]
When I try to group match with regexp like we do on VScode:
![image](https://user-images.githubusercontent.com/12798126/128532132-a5673a0c-4c4b-4db1-bf3a-32e7dba35326.png)

It matches the group by () and $1 syntax. However, on Theia 1.16.0, the result is like this:
![image](https://user-images.githubusercontent.com/12798126/128532745-10dc8e81-64e4-41f6-9c36-f4d264ed91b4.png)

Am I forgetting something on my theia build? Any helps are appreaciated 🙏

**Comment by vince-fugnitto** [comment]
@furkanarabaci thank you for the discussion, I believe it is simply not yet supported and up for grabs: https://github.com/eclipse-theia/theia/issues/6420.
